lib/kennel/syncer.rb

```diff
@@ -221,6 +221,14 @@ def print_plan(step, list, color)
     end
 
     def print_diff(diff)
+      max_diff_lines = [Integer(ENV.fetch("MAX_DIFF_LINES", "50")), 10].max
+      require_relative "./output_limiter"
+      out = OutputLimiter.new(Kennel.out, max_diff_lines) do
+        message = "(Diff for this item truncated after #{max_diff_lines} lines. " \
+          "Rerun with MAX_DIFF_LINES=#{max_diff_lines * 2} to see more)"
+        Kennel.out.puts "  " + Utils.color(:magenta, message)
+      end
+
       diff.each do |type, field, old, new|
```

Comment:
maybe we build a string here instead
... and then have a `truncate text, to: 100, with: "text was truncated blah blah"` method ?

Reply:
Code Golf later :-)